portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomPoem() {
   const poemContainer = document.getElementById('poem-container');
   poemContainer.innerText = poem;
 }
+
+async function showInterest(){
```

Comment:
Minor nit: add a space before the opening brace.
ie. `async function showInterest() {`